### ISSUE TITLE
Show filters only when JS is enabled

### ISSUE THREAD

**Issue by rhernand3z** [NYPL/staff-picks#52]
The left sidebar filters should ONLY be displayed if JS is enabled. Another PR tracked the work to pass the boolean that checks if JS is enabled, once that is merged this task will focus on:

- Initially hiding the filters section when rendered server-side by Node
- Once the React mounts, the sidebar section will get an updated `isJsEnabled` boolean and ONLY render the filters if the value is `true`.
- A loading transition might need to be implemented -- @ricardoom or we can simply fade it in...

CC: @RobKelley

**Comment by rhernand3z** [comment]
Fixed by #82
